src/arch/x86/reg/id.rs

```diff
@@ -1,5 +1,50 @@
+use crate::arch::RegId;
+
 // TODO: Add proper `RegId` implementation. See [issue #29](https://github.com/daniel5151/gdbstub/issues/29)
 // pub enum X86RegId {}
```

Comment:
If it's not too much trouble, could you also implement the X86RegId as well?

Feel free to say no -- it's just that the two implementations aught to be fairly similar (I'd imagine), and it'd be nice to get both of them out of the way at the same time.

Reply:
Sure!